db/migrate/20240713130357_add_token_counts_to_messages.rb

```diff
@@ -0,0 +1,6 @@
+class AddTokenCountsToMessages < ActiveRecord::Migration[7.1]
+  def change
+    add_column :messages, :input_token_count, :integer
```

Comment:
I think this probably needs a default: 0 and a null: false. Probably just the null: false is what’s required, but I think it’s worth having default 0 too: this could make a difference when we’re doing a SQL query like `…sum(:input_token_count)` because activerecord won’t instantiate each record